apps/client/src/components/Router.js

```diff
@@ -1,23 +1,26 @@
-import React from "react";
-import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import React, { Suspense } from "react";
```

Comment:
```Suspense for Data Fetching (Experimental);```

Should we use experimental things in production?

Reply:
Not sure... It says: "React 16.6 added a <Suspense> component that lets you “wait” for some code to load and declaratively specify a loading state (like a spinner) while we’re waiting"
I think the experimental feature is waiting for data instead of code/components. Also we use a stable version of react and not the experimental build like it states in https://reactjs.org/docs/concurrent-mode-suspense.html .
What do you think, can we use this feature?